direct/core/src/test/java/cz/o2/proxima/direct/storage/InMemStorage.java

```diff
@@ -1062,7 +1070,7 @@ public DataAccessor createAccessor(
             .map(v -> Integer.parseInt(v.toString()))
             .orElse(1);
 
-    final Partitioner partitioner = new KeyAttributePartitioner();
+    final Partitioner partitioner = new KeyPartitioner();
```

Comment:
Why did we change the default? The default in other commit-logs is KeyAttribute partitioner.

Reply:
Let me see if change breaks anything. There was a motivation around simpler partition for tests (where you need to know, where your element ends up, before you write it), but that's something we can work around with proper tooling.